.travis.yml

```
@@ -5,8 +5,8 @@ jdk: openjdk8
 
 env:
   global:
-    BYOND_MAJOR="512"
-    BYOND_MINOR="1479"
+    BYOND_MAJOR="513"
+    BYOND_MINOR="1534"
```

Or have a quick shortcut:
```suggestion
    BYOND_MINOR="1536"
```